components/TestLibraryStepper/TestLibraryStepper.tsx

```diff
@@ -0,0 +1,147 @@
+﻿import React, {useState} from "react";
+import {Step, StepConnector, StepIconProps, StepLabel, Stepper, Typography} from '@material-ui/core';
+import {createMuiTheme, makeStyles, MuiThemeProvider} from '@material-ui/core/styles';
+import Router from "next/router";
+import {withStyles} from "@material-ui/styles";
+import clsx from "clsx";
+import {Check, MoreHoriz} from "@material-ui/icons";
+import {CandidateTestStatus} from "../../pages/api/candidateApiClient.module";
+
+interface TestLibraryStepperProps {
+    candidateTestStatus: CandidateTestStatus[];
+}
+
+function getSteps() {
+    //TODO: mock data, set number of steps based on test range, maybe props.candidateTestStatus.tests.length?
+    const testNumber = 3;
+
+    //set labels for steps
+    const testLabelArray = [];
+    while (testLabelArray.length + 1 <= testNumber) {
+        testLabelArray.push(`Test ${testLabelArray.length + 1}`);
+    }
+    return testLabelArray;
+}
+
+export default function TestLibraryStepper(props: TestLibraryStepperProps): JSX.Element {
+    const steps = getSteps();
```

Comment:
could we change this so that it uses what's passed in by the props?

Reply:
ie I think this is now just `const steps = props.map(test => test.title)`

Reply:
I put in the props instructions in a comment as it won't work without dummy data. Hope that works until after endpoint is live?

Reply:
I need to keep the getSteps() function to set the material ui labels for the steps unfortunately ;(

Reply:
fine for now - in the future I generally find that a better pattern is to put the fake data in as high up as possible.

ie in this case, I'd make this component like the totally real version, but then in the page I'd just send it fake data rather than the real thing.

That way, when I come to switch out the fake data for real data, I've only got 1 thing to change, and everything else just works